xapian-bindings/python3/README

```diff
@@ -0,0 +1,29 @@
+Building a Python package
+=========================
+
+First, build xapian-core for static linking:
+
+$ cd xapian-core
+$ ./configure --enable-static --disable-shared --enable-pic [--enable-maintainer-mode] (when building from git)
+$ make
```

Comment:
If you're building from git, we strongly recommend using the top-level configure as that knows how to hook up the modules to build against the in-tree xapian-core and how to avoid other pitfalls, so I wouldn't want to suggest other approaches.

I'm also not sure I want to start encouraging people to build stable versions from git.  You need more tools installed and there's more that can go wrong so it can be a less smooth experience compared to building from a release tarball.

Without the rest of the packaging info, documenting how to build xapian-core statically such that it can be linked into a shared object is perhaps of use, but really doesn't belong in the docs for the python bindings.